cmake/bintray_deploy.json.in

```diff
@@ -0,0 +1,28 @@
+{
+    "package": {
+        "name": "Cockatrice-dev",
+        "repo": "Cockatrice",
+        "subject": "cockatrice",
+        "desc": "Cockatrice master branch automated builds",
+        "website_url": "https://github.com/Cockatrice/Cockatrice",
+        "issue_tracker_url": "https://github.com/Cockatrice/Cockatrice/issues",
+        "vcs_url": "https://github.com/Cockatrice/Cockatrice.git",
+        "github_use_tag_release_notes": true,
+        "github_release_notes_file": "RELEASE.txt",
+        "licenses": ["GPL-2.0"],
+        "labels": ["card", "tabletop", "game"],
+        "public_download_numbers": false,
+        "public_stats": false
+    },
+    "version": {
+        "name": "@PROJECT_VERSION@",
+        "desc": "Unstable builds from master",
+        "vcs_tag": "@GIT_COMMIT_ID@",
+        "gpgSign": false
```

Comment:
Yep, i plan "advanced bintray" for a next PR (debian repo on bintray, gpgsign, ...)